theme6/source/partials/_footer.erb

```diff
@@ -0,0 +1,40 @@
+<div class="contenedor">
+      <footer>
+          <div class="contenedorsocial">
```

Comment:
Let's make class names more readable by using `-` to separate words; `contenedorsocial` => `contenedor-social`